packages/graphiql/src/new-components/Toolbar/Tabs.js

```diff
@@ -0,0 +1,48 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui';
+import PropTypes from 'prop-types';
+import WithDividers from './support/WithDividers';
+
+const Tab = ({ active, ...props }) => (
+  <button
+    sx={{
+      padding: ({ spaces }) => spaces.rowPadding,
+      outline: 'none',
+      textAlign: 'end',
+      verticalAlign: 'baseline',
+      transition: ({ transitions }) => transitions[0],
+      cursor: 'pointer',
+      ':focus, :hover': {
+        boxShadow: active ? 'primaryUnderline' : 'underline',
+        color: active ? 'primary' : 'darkText',
+      },
+      boxShadow: active ? 'primaryUnderline' : 'inset 0 0 0 transparent',
+      color: active ? 'primary' : 'text',
+    }}
+    {...props}
+  />
+);
+Tab.propTypes = { active: PropTypes.bool };
+
+const Tabs = ({ tabs, active, onChange }) => {
+  return (
+    <WithDividers>
+      {tabs.map((tab, index) => (
+        <Tab
+          key={index}
+          active={active === index}
+          onClick={() => onChange(index)}>
```

Comment:
eventually we might want a readable ID here, but i think that would be up to the particular reducer action that fires when `onChange` happens, not this business-agnostic component.